aws/modules/consolidate-logs/module.ftl

```diff
@@ -0,0 +1,226 @@
+[#ftl]
+
+[@addModule
+    name="consolidatelogs"
+    description="Solution-wide consolidation of logs, intended for consumption by ElasticSearch."
+    provider=AWS_PROVIDER
+    properties=[
+        {
+            "Names" : "namePrefix",
+            "Type" : STRING_TYPE,
+            "Description" : "A prefix appended to component names and deployment units to ensure uniquness",
+            "Default" : "log"
+        },
+        {
+            "Names" : "deploymentProfile",
+            "Type" : STRING_TYPE,
+            "Description" : "The deployment profile id to enable log consolidation on."
```

Comment:
Should either add a `Default` or `Manadatory` flag her

Reply:
I've gone with `Mandatory` as it feels safer to have the user define what they want than have them only half-evaluate if any given default group is appropriate